### PR TITLE
STYLE: Remove superfluous calls to "find_package(Slicer ...)"

### PR DESCRIPTION
**Pull request by jcfr**

### CODE REVIEW

File: DoseVolumeHistogram/SubjectHierarchyPlugins/qSlicerSubjectHierarchyDoseVolumeHistogramPlugin.cxx [hunk 1]
```diff
@@ -349,13 +349,13 @@ vtkIdType qSlicerSubjectHierarchyDoseVolumeHistogramPlugin::getChartShItemForDvh
   if (!scene)
   {
     qCritical() << Q_FUNC_INFO << ": Invalid MRML scene";
-    return NULL;
+    return 0;
```
Comment (cpinter) [inline DoseVolumeHistogram/SubjectHierarchyPlugins/qSlicerSubjectHierarchyDoseVolumeHistogramPlugin.cxx:352]: Why are these changes necessary? NULL is everywhere in SlicerRT (and in Slicer) and it works well. Actually in SlicerRT I consider this part of the coding convention.

Comment (jcfr) [reply DoseVolumeHistogram/SubjectHierarchyPlugins/qSlicerSubjectHierarchyDoseVolumeHistogramPlugin.cxx:352]: It was reporting a warning. 0 if for number, NULL for pointer.

Comment (lassoan) [reply DoseVolumeHistogram/SubjectHierarchyPlugins/qSlicerSubjectHierarchyDoseVolumeHistogramPlugin.cxx:352]: For me, 0 is as bad as NULL (but with NULL the coder's intent is a bit more clear). Fortunately, nullptr is introduced in cxx11 - we should just switch to that.

Comment (jcfr) [reply DoseVolumeHistogram/SubjectHierarchyPlugins/qSlicerSubjectHierarchyDoseVolumeHistogramPlugin.cxx:352]: To clarify, the return type is `vtkIdType`, I don't think `nullptr` would work either.

Comment (lassoan) [reply DoseVolumeHistogram/SubjectHierarchyPlugins/qSlicerSubjectHierarchyDoseVolumeHistogramPlugin.cxx:352]: Sure, if it's vtkIdType then it should be 0. Or, even nicer: vtkMRMLSubjectHierarchyNode::INVALID_ITEM_ID

Comment (cpinter) [reply DoseVolumeHistogram/SubjectHierarchyPlugins/qSlicerSubjectHierarchyDoseVolumeHistogramPlugin.cxx:352]: Ah OK. If it's a number, then of course it should be 0. vtkMRMLSubjectHierarchyNode::INVALID_ITEM_ID was the original constant for invalid SH ID, but then we changed it to be 0 just because it gives more concise, easier to read (and write) code. 0 is fine with me!

If it was a pointer, then my feelings coincide with Andras'. I use NULL mostly for clarity (it's then obvious it's a pointer). I also agree with the eventual shift to nullptr.

In summary no need to change anything here :) Thanks for the answers
